react-function-components/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+root.render(CustomButton());
```

Comment:
We don't want to call our function component, instead create a react element `<CustomButton />`